### ISSUE TITLE
Unable to retrieve any error message on EncounteredError event

### ISSUE THREAD

**Issue by jedesire** [ZeBobo5/Vlc.DotNet#429]
I have a question about Vlc.DotNet.

# Generic information

- **Vlc.DotNet version** : 2.2.1
- **Vlc.DotNet project used** : WPF
- **libvlc version** : 3.0.3 (tested with both x86/x64) [from VLC's install folder]
- **.net version** : 4.6.2
- **Project language** : C#
- **Project build architecture** : AnyCPU
- **Operating system** : Windows 10 x64

# Summary

I'm developing a media player in MVVM. For this, I use a ContentPresenter with a Binding to my VlcControl, which I instantiate in the ctor :
```
m_mediaPlayer = new VlcControl();
m_mediaPlayer.MediaPlayer.VlcLibDirectoryNeeded += OnVlcLibDirectoryNeeded;
m_mediaPlayer.MediaPlayer.EndInit();
m_mediaPlayer.MediaPlayer.EncounteredError += MediaPlayer_EncounteredError;
```

Now, I'm testing it with USB stick. I play a file from it, and then unplug it. 
I can see that EncounteredError is raised, but I'm not able to retrieve any error message, which is bad because I need to display an error message to the user.

Could you please help me ?
Thanks !

**Comment by jeremyVignelles** [comment]
Yes, EncounteredError does not provide an error message. That's the way libvlc is. However, since Vlc.DotNet 3.0 (still in beta), I added the Dialogs API, which you can use to display an error to the user (as VLC would).

See : https://github.com/ZeBobo5/Vlc.DotNet/tree/develop/src/Samples/Samples.Wpf.Dialogs

**Comment by jedesire** [comment]
Thanks for you quick answer !

What are the current limitations of the beta 3 ?
Do you have any idea (roughly) of an official release ?

**Comment by jeremyVignelles** [comment]
I think that the 3.0 -develop is as stable as the 2.x, but those things has changed, and I wanted to make sure not to break something:
- I became the maintainer of this project. I made the develop branch and develop packages
- SEMVER was applied
- A new WPF control : the most breaking change (see the readme). I wanted to make sure the API was correct.
- Several bugs were fixed

Now that those things are done, there is sill 1-2 bugs and pull requests to merge, but I don't expect the API to change in a significant way for 3.0.

As for an estimation, it varies from 2 weeks to 1-2 month. it depends on how much time I have available.
I wouldn't expect that to happen before june the 21st, when I'll be back from holidays
